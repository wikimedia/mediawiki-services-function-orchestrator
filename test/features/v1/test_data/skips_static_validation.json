{
	"Z1K1": "Z7",
	"Z7K1": {
		"Z1K1": "Z8",
		"Z8K1": [
			"Z1"
		],
		"Z8K2": "Z1",
		"Z8K3": [
			"Z1"
		],
		"Z8K4": [
			"Z14",
			{
				"Z1K1": "Z14",
				"Z14K1": "Z1248",
				"Z14K2": {
					"Z1K1": "Z7",
					"Z7K1": "Z801",
					"Z801K1": {
						"Z1K1": "Z40",
						"Z40K1": "NOT A BOOLEAN AT ALL"
					}
				}
			}
		],
		"Z8K5": "Z1248"
	}
}
