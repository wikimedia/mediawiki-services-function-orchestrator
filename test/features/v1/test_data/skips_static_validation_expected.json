{ "Z1K1": "Z5", "Z5K1": { "Z1K1": "Z518", "Z518K1": "Z40", "Z518K2": { "Z1K1": "Z40", "Z40K1": "NOT A BOOLEAN AT ALL" }, "Z518K3": { "Z1K1": "Z5", "Z5K1": "Z502", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z502" }, "Z502K1": "Z509", "Z502K2": { "Z1K1": "Z5", "Z5K1": "Z509", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z509" }, "Z509K1": [ { "Z1K1": "Z5", "Z5K1": "Z526", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z526" }, "Z526K1": { "Z1K1": "Z39", "Z39K1": "Z40K1" }, "Z526K2": { "Z1K1": "Z5", "Z5K1": "Z535", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z535" }, "Z535K1": { "Z1K1": "Z99", "Z99K1": "NOT A BOOLEAN AT ALL" } } } } }, { "Z1K1": "Z5", "Z5K1": "Z535", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z535" }, "Z535K1": { "Z1K1": "Z99", "Z99K1": { "Z1K1": "Z40", "Z40K1": "NOT A BOOLEAN AT ALL" } } } }, { "Z1K1": "Z5", "Z5K1": "Z511", "Z5K2": { "Z1K1": { "Z1K1": "Z7", "Z7K1": "Z885", "Z885K1": "Z511" }, "Z511K1": { "Z1K1": "Z39", "Z39K1": "Z18K1" }, "Z511K2": { "Z1K1": "Z99", "Z99K1": { "Z1K1": "Z40", "Z40K1": "NOT A BOOLEAN AT ALL" } } } } ] } } } } } }
